src/raspi/state_management/utils/logger.py

```diff
@@ -30,11 +32,22 @@ def configure_logger(level: str = "Debug"):
     """
     lvl = map_level(level)
     print(f"Configuring logger {lvl}...")
-    start_time = d.now().strftime("%Y-%m-%d.%H:%M:%S")
-    filename = __main__.__file__.split("/")[-1].split(".")[0]
-    logging.basicConfig(
-        filename=f".log/{start_time}.{filename}.{level}.log",
-        format="%(filename)s: %(message)s",
-        level=lvl,  # TODO: hook it to env or config file
-        force=True,
-    )
+    if is_dev():
+        print("Using dev logger")
+        start_time = d.now().strftime("%Y-%m-%d_%H-%M-%S")
+        logging.basicConfig(
+            filename=f".log/{start_time}.DEV.{level}.log",
+            format="%(filename)s: %(message)s",
+            level=lvl,  # TODO: hook it to env or config file
+            force=True,
+        )
+    else:
+        print("Using regular logger")
+        start_time = d.now().strftime("%Y-%m-%d.%H:%M:%S")
+        filename = __main__.__file__.split("/")[-1].split(".")[0]
+        logging.basicConfig(
+            filename=f".log/{start_time}.{filename}.{level}.log",
+            format="%(filename)s: %(message)s",
+            level=lvl,  # TODO: hook it to env or config file
+            force=True,
+        )
```

Comment:
What is the benefit of this?

Reply:
The normal logger file name causes a crash on our local machine because it uses invalid characters (colons). Also, including the "filename" variable in the file name causes the entire file path to be copied into the name, including /'s. I don't know if there's any importance for the loggers to use the old file name when running on the pi, so I kept that there just in case.

Reply:
oh I did not realize this was a thing lol. You can make this a default for both so its the same convention on every device.